### PR TITLE
Add app locale settings

### PR DESCRIPTION
**Pull request by nohli**
As mentioned in [this PR](https://github.com/spencerccf/app_settings/pull/179#issuecomment-1628639895)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,10 @@ All three options open the current 'app' settings section if there are settings
 ### Android
 Each option will open and display the corresponding screen: WIFI, Location, or Security, etc.
 
-## 5.0.0
+## 4.4.0
+Add `AppSettingsType.appLocale` to open the app language settings (only available on Android 13 and higher).
+
+## 4.3.0
```
Comment (nohli) [inline CHANGELOG.md:13]:
```suggestion
## 5.1.0
Add `AppSettingsType.appLocale` to open the app language settings (only available on Android 13 and higher).

## 5.0.0
```

File: pubspec.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: app_settings
 description: A Flutter plugin for opening iOS and Android phone settings from an app.
-version: 5.0.0
+version: 4.4.0
```
Comment (nohli) [inline pubspec.yaml:3]:
```suggestion
version: 5.1.0
```

File: android/src/main/kotlin/com/spencerccf/app_settings/AppSettingsPlugin.kt [hunk 3]
```diff
@@ -156,6 +157,23 @@ class AppSettingsPlugin: FlutterPlugin, MethodCallHandler, ActivityAware {
     result.success(null)
   }
 
+  /**
+   * Open the application's preferred locale settings.
+   */
+  private fun openAppLocaleSettings(result: Result, asAnotherTask: Boolean = false) {
```
Comment (navaronbracke) [inline android/src/main/kotlin/com/spencerccf/app_settings/AppSettingsPlugin.kt:163]: Could you add a check for the Android version and only do the `startActivity()` on Android 13+ ? That way it does not throw on unsupported Android versions.

**Review by navaronbracke — CHANGES_REQUESTED** [review]

**Review by navaronbracke — APPROVED** [review]

**Comment by nohli** [comment]
Are there plans for merging, or do I need to change anything?

**Comment by navaronbracke** [comment]
I do not have commit access, so it is up to @spencerccf to review and merge this, sorry.

You can always use a git override in the meantime, though.

**Comment by nohli** [comment]
Ok thanks 🤗

That's what I'm doing, just checking up on some PRs 😊

**Comment by nohli** [comment]
Would be cool to publish to pub.dev as well.

**Pull request merged**